daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md

```diff
@@ -702,7 +702,7 @@ go build app.go
 Run the `order-processor` subscriber service alongside a Dapr sidecar.
 
 ```bash
-dapr run --app-port 6002 --app-id order-processor --app-protocol http --dapr-http-port 3501 --components-path ../../../components -- go run app.go
+dapr run --app-port 6001 --app-id order-processor --app-protocol http --dapr-http-port 3501 --components-path ../../../components -- go run app.go
```

Comment:
The problem here is with the code. Validate for me, but the Go HTTP sample uses 6002 and the SDK sample uses 6001. They should both be the same otherwise fixing this docs issue with break the other one. so this is a code bug.

Reply:
since the quickstart focuses on the SDK, ok to update this to 6001? ...should we mention somewhere to follow the HTTP READMEs for the correct commands? at least until we make the code more consistent?

Reply:
We should fix the code to be the same.

Reply:
This still seems to be broken, is there an issue captured for QS to change these ports to be consistent?

Reply:
> This still seems to be broken, is there an issue captured for QS to change these ports to be consistent?

I put this issue into this issue https://github.com/dapr/quickstarts/issues/656 and marked this as P0.